src/models/products.Category.Model.ts

```diff
@@ -0,0 +1,38 @@
+import { DataTypes, Model } from 'sequelize'
+import { development, production, testing } from '../db/config'
+
+const isProduction = process.env.NODE_ENV === 'production'
+const isTesting = process.env.NODE_ENV === 'testing'
+const sequelize = isProduction ? production : isTesting ? testing : development
+
+class Category extends Model {
+  categoryId!: string
+  name!: string
+  description!: string
+}
+
+Category.init(
```

Comment:
@Aldot-02  here table relations are needed, beacause category should be related with product as well as user, make some researches, but one way i have read is to make references using migrations